lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -372,7 +372,7 @@ def get_enb_s1_connected_states(configured_serial_ids, mconfig) -> List[State]:
                                   mme_connected=True,
                                   fsm_state='N/A')
             status_dict = status._asdict()
-            status_dict['ip_address'] = enb_config.ip_address
+            status_dict['ip_address'] = enb_config.config.ip_address
```

Comment:
Sanity check that this is in fact supposed to be `config.config`? Looks a bit weird.